meta-resin-common/recipes-devtools/kernel-modules-headers/kernel-modules-headers.bb

```diff
@@ -46,7 +46,7 @@ do_compile() {
         fi
     done
 
-    tar -czf kernel_modules_headers.tar.gz kernel_modules_headers
```

Comment:
Why don't we create this archive directly into deploy directory? Looks like an unnecessary dance to create it in B and afterwards move it to deploy directory.

Reply:
More of semantic thing.
Technically, do_compile shouldn't really be touching anything in the deploy directly.

I guess we can tarball once directly into the deploy directly in do_deploy

Reply:
That makes sense. Let's leave it for now.